packages/ui/src/components/Header/Header.tsx

```diff
@@ -1,33 +1,58 @@
+'use client'
 import { Icon } from '../Icon'
 import { twMerge } from 'tailwind-merge'
 
 export interface HeaderProps {
+  page?: string
   greeting?: string
   name: string
   textStyle?: string
   iconStyle?: string
+  isHelperIcon?: boolean
```

Comment:
Same goes here, you won't be getting any prop such as isHelperIcon from the json. You should render based out of the path name.